sendEmail.js

```diff
@@ -4,3 +4,96 @@
 
 // Let the user know if the passwords to not match
 // Also let the user know when the email has been successfully sent
+
+let fname;
```

Comment:
Would be better to have these variables local to the function instead of reusing them in different functions.  It looks like fname is a string in the main function but transforms into an object in the button event listener